jellyfin-platform-android/src/main/kotlin/org/jellyfin/apiclient/interaction/AndroidDevice.kt

```diff
@@ -9,13 +9,14 @@ import org.jellyfin.apiclient.model.DeviceInfo
 /**
  * Helper function used in [androidDevice] to normalize device names:
  *
- * - Removes special characters from the name.
+ * - Removes non US-ASCII printable characters from the name.
```

Comment:
I believe there's no other ASCII, so the extra US seems redundant.

```suggestion
 * - Removes non-ASCII-printable characters from the name.
```

Reply:
It is the [encouraged name](https://www.iana.org/assignments/character-sets/character-sets.xhtml#character-sets-1:~:text=The%20use%20of%20the%20name%20US%2DASCII,encouraged.) established by the IANA so that it is not confused with the multiple [variants and derivations](https://en.wikipedia.org/wiki/ASCII#Variants_and_derivations) that exist of the mislabelled ASCII, but I can change it to ASCII if you want

Reply:
Interesting! Didn't know, that's probably more factual then. I'd still add the additional hyphens though.

Reply:
> I'd still add the additional hyphens though.

So `Removes non-US-ASCII-printable characters from the name.`?

Reply:
Yup.